test/fixtures/mson/enum-variants.json

```diff
@@ -59,6 +59,17 @@
                       },
                       "value": {
                         "element": "enum",
+                        "attributes": {
+                          "enumerations": {
+                            "element": "array",
+                            "content": [
+                              {
+                                "element": "string",
+                                "content": "L"
+                              }
+                            ]
```

Comment:
This matches any `string`. Are you missing `fixed` attribute on the `string` elements above?